package.json

```diff
@@ -42,5 +45,12 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-commit": "lint-staged"
+    }
   }
```

Comment:
I don't think this is needed here. We have `.husky` folder with all the githooks